### ISSUE TITLE
War Fervor event

### ISSUE THREAD

**Issue by Der-Zorn-Gottes** [LoneGazebo/Community-Patch-DLL#1827]
Bug Report Template

Mod Version (i.e Date - (4/23b)): 5/14

1. Mod List (if using standard CPP set, leave blank):

2. Type of error (i.e. crash, interface bug, AI quirk): 

Of the five options you have, 4 are limited to 90 turns, but the one that gives +3 production isn't. I don't know if it is not limited, or if it just isn't mentioned in the description.

3. Steps to reproduce:

4. Additional information:

---------------------------
Supporting information:

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
2. Minidump file (located in your Civ5.exe directory)
3. Screenshots (if needed)

**Comment by Edaka** [comment]
Also, after choosing the option that gives +5 happiness and waiting for it to expire, the notification about the expiration seemed not to show what event had just expired. I only had one active event at the time, so I knew it was that one. I'll post a related screenshot when I can.

**Comment by Edaka** [comment]
Checked both the PlayerEvents.sql and the text file - the event choice you're talking about has  a duration of 30 turns on standard speed (in fact, all of the war fervor choices do), and the text includes that info.  I'm also pretty sure that one time I got the event and made that choice, I got informed about the duration.

Regarding my comment about event expiration notifications: I see now there's `TXT_KEY_NOTIFICATION_EVENT_HAS_ENDED_CITY` and `TXT_KEY_NOTIFICATION_EVENT_HAS_ENDED_CITY_T` , two different strings serving what seems to be one purpose. Why are they different? Which purpose does each of them serve and when is each used?
